### PR TITLE
Overriding lr scheduler in torchvision

### PR DESCRIPTION
**Pull request by corey-nm**

### CODE REVIEW

File: src/sparseml/pytorch/torchvision/train.py [hunk 1]
```diff
@@ -436,6 +436,9 @@ def collate_fn(batch):
     manager = ScheduledModifierManager.from_yaml(args.recipe_path)
     optimizer = manager.modify(model, optimizer, len(data_loader))
 
+    if manager.learning_rate_modifiers:
+        lr_scheduler = None
```
[the remark this answers is not in the document]
Comment (dbogunowicz) [reply src/sparseml/pytorch/torchvision/train.py:440]: We definitely need (manual) testing for this PR.

**Review by dbogunowicz — APPROVED** [review]
Please add the description to the PR and plan on how to test this now/in the future.

File: src/sparseml/pytorch/torchvision/train.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (corey-nm) [reply src/sparseml/pytorch/torchvision/train.py:440]: do we want people going through the cli for lr scheduler? the recipe needs to know if lr is getting changed right?

[the remark this answers is not in the document]
Comment (bfineran) [reply src/sparseml/pytorch/torchvision/train.py:440]: no, we don't want people running through - preferred method is to use the recipe. my point was that line 416 expects there to be something in the state dict to load for the scheduler, just need to account for the fact that in our saved models it may not be there :)

[the remark this answers is not in the document]
Comment (bfineran) [reply src/sparseml/pytorch/torchvision/train.py:440]: bump on this

[the remark this answers is not in the document]
Comment (corey-nm) [reply src/sparseml/pytorch/torchvision/train.py:440]: Gonna close this PR and open a different one after checkpoints are merged in - will address this in that one
